src/node/nodestate.h

```diff
@@ -734,14 +745,24 @@ namespace ccf
         *recovery_signature_map,
         *recovery_nodes_map);
 
-      recovery_encryptor =
+      recovery_encryptor = nullptr;
 #ifdef USE_NULL_ENCRYPTOR
-        std::make_shared<NullTxEncryptor>();
+      recovery_encryptor = std::make_shared<NullTxEncryptor>();
 #else
-        // Recovery encryptor should not seal ledger secrets on compaction.
-        // Since private ledger recovery is done in a temporary store, ledger
-        // secrets are only sealed once the recovery is successful.
-        std::make_shared<TxEncryptor>(self, network.ledger_secrets, true);
+      // Recovery encryptor should not seal ledger secrets on compaction.
+      // Since private ledger recovery is done in a temporary store, ledger
+      // secrets are only sealed once the recovery is successful.
+
+      // if consensus is pbft use null encryptor
```

Comment:
I think this should get the comment from CMake, explaining _why_ we're using the null encryptor rather than re-describing the code:

```
# for now do not encrypt the ledger as the current implementation does not	
# work for PBFT
```